### PR TITLE
WK2-943 Added information about fee process in prod/sandbox

### PR DESCRIPTION
**Pull request by tom-meader-wk**
# Checklist

- [x] I have spellchecked my changes.
- [x] I have updated the API Documentation CHANGELOG where applicable.
- [x] I have added my guides to Welcome.md where applicable
- [x] I have added my guides to toc.json where applicable.

### CODE REVIEW

File: docs/sandbox/Sandbox-FAQ.md [hunk 1]
```diff
@@ -72,3 +72,12 @@ Currently not all of the securities we have set up in Production are mirrored in
 
 Similarly, we do not have up to date prices in Sandbox, and the ones that are generated each day stay the same for each ISIN.
 
+### Fees
+
+#### Sandbox
+
+Fee accrual, aggregation and charging is manual in sandbox, so if you want to test out fees, you'll need to contact your tenant manager. We will need the fee portfolio that you've set up and the accrual strategy details that you've agreed with your tenant manager. Typically we will simulate this for the previous month.
```
Comment (niall-maloney) [inline docs/sandbox/Sandbox-FAQ.md:79]:
```suggestion
Fee accrual, aggregation and charging is manual in sandbox. If you would like to test out fees in Sandbox please contact your tenant manager. 

To test in sandbox we will need the ID of the fee portfolio that you have set up and the accrual strategy details that have been agreed upon with your tenant manager. Typically we will simulate this for the previous month but let your tenant manager know if you want a different period.
```

File: docs/sandbox/Sandbox-FAQ.md [hunk 2]
```diff
@@ -72,3 +72,12 @@ Currently not all of the securities we have set up in Production are mirrored in
...
+
+#### Production
+
+In production this process is automated, with the only manual step being the payment out once fees have been charged.
```
Comment (niall-maloney) [inline docs/sandbox/Sandbox-FAQ.md:83]:
```suggestion
In production this process is automated.
```

File: docs/sandbox/Sandbox-FAQ.md [hunk 3]
```diff
@@ -72,3 +72,12 @@ Currently not all of the securities we have set up in Production are mirrored in
 
 Similarly, we do not have up to date prices in Sandbox, and the ones that are generated each day stay the same for each ISIN.
 
+### Fees
```
Comment (niall-maloney) [inline docs/sandbox/Sandbox-FAQ.md:75]: Should we order these alphabetically?

File: docs/sandbox/Sandbox-FAQ.md (hunk 1 above)
Comment (LukeBillo) [reply docs/sandbox/Sandbox-FAQ.md:79]: I wouldn't bother mentioning "aggregation" - the idea of fees being aggregated is foreign to our tenants, it's an implementation detail.
Just fee accrual and charging is fine.

Comment (LukeBillo) [reply docs/sandbox/Sandbox-FAQ.md:79]: Is it worth mentioning that if they want the fees to be charged, they'll need to put enough cash in the portfolio themselves? (either via deposit or sell orders)

**Pull request merged**